Update new-relic-api-keys.mdx (#20163)
newrelic/docs-website: src/content/docs/apis/intro-apis/new-relic-api-keys.mdx

## View and manage API keys [#keys-ui]

You can manage most API keys from the [API Keys UI page](https://one.newrelic.com/api-keys) located in the [user menu](/docs/accounts/accounts-billing/general-account-settings/intro-account-settings/#user-menu) when you click <DNT>**API keys**</DNT>.
You can also [use NerdGraph to manage keys](/docs/apis/nerdgraph/examples/use-nerdgraph-manage-license-keys-user-keys)

We temporarily provide to you the entire key during the creation of it, but after creating it we only provide the first 8 characters of the key in the API Keys UI.

> If after key creation you need to obatin the full key again, you would click on the ellipses in the API Keys UI, copy the key ID and then [use NerdGraph to query for the full key ](/docs/apis/nerdgraph/examples/use-nerdgraph-manage-license-keys-user-keys/#query-keys).

Reviewer: ```suggestion
If you need to retrieve the full key after creation, click the ellipses in the API Keys UI, copy the key ID, and [use NerdGraph to query for the full key.](/docs/apis/nerdgraph/examples/use-nerdgraph-manage-license-keys-user-keys/#query-keys).
```
